.claude/agents/pr-scope-checker.md

```
@@ -0,0 +1,195 @@
+decision interlocked
+```
+
+→ 진행.
+
+---
+
+### SUGGEST_SPLIT
+
+조건 하나 이상 해당:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)